rmgpy/molecule/atomtypeTest.py

```diff
@@ -276,9 +586,35 @@ def testSulfurTypes(self):
         """
         Test that getAtomType() returns appropriate sulfur atom types.
         """
-        self.assertEqual(self.atomType(self.mol4, 8), 'Ss')
-        self.assertEqual(self.atomType(self.mol4, 10), 'Sd')
-        self.assertEqual(self.atomType(self.mol21, 1), 'St')
+        self.assertEqual(self.atomType(self.mol22, 0), 'Sa')
+        self.assertEqual(self.atomType(self.mol21, 0), 'S0c')
+        self.assertEqual(self.atomType(self.mol23, 0), 'S2s')
+        self.assertEqual(self.atomType(self.mol21, 1), 'S2sc')
+        self.assertEqual(self.atomType(self.mol42, 2), 'S2sc')
+        self.assertEqual(self.atomType(self.mol19, 1), 'S2d')
+        self.assertEqual(self.atomType(self.mol24, 1), 'S2dc')
+        self.assertEqual(self.atomType(self.mol69, 1), 'S2tc')
+        self.assertEqual(self.atomType(self.mol25, 0), 'S4s')
+        self.assertEqual(self.atomType(self.mol23, 1), 'S4sc')
+        self.assertEqual(self.atomType(self.mol25, 2), 'S4d')
+        self.assertEqual(self.atomType(self.mol28, 1), 'S4dd')
+        self.assertEqual(self.atomType(self.mol38, 1), 'S4dd')
+        self.assertEqual(self.atomType(self.mol26, 1), 'S4dc')
+        # self.assertEqual(self.atomType(self.mol27, 0), 'S4b')  # Not yet implemented. See RMG-Py issue #982
```

Comment:
This appears to be implemented, so should the comment say that it doesn't work properly instead?

Reply:
changed